integrationtests/src/test/java/io/kroxylicious/proxy/multitenant/MultiTenantIT.java

```diff
@@ -289,6 +298,118 @@ public void describeGroup(KafkaCluster cluster) throws Exception {
         }
     }
 
+    @Test
+    public void produceInTransaction(KafkaCluster cluster) throws Exception {
```

Comment:
Unlike the production code I do think we should split off a separate `TransactionalMutliTenantIT` as a more focused class which could potentially standardise some common config for producers and consumers.

Reply:
Yeah, that's a good point.

Reply:
To avoid generating additional work for other PRs, I'd prefer to make this change separately.

Reply:
Raised as https://github.com/kroxylicious/kroxylicious/issues/349